src/Xml/Decode.elm

```diff
@@ -97,8 +97,14 @@ parseSlice first firstClose trimmed =
             [] ->
                 if String.startsWith "?" tagName then
                     Ok ( DocType tagName props, firstClose + 1 )
-                else if (String.contains "/>" trimmed) then
-                    Ok ( Tag tagName props (Object []), firstClose + 1 )
+                else if String.endsWith "/" beforeClose then
+                    let
+                        tag = if String.endsWith "/" tagName then
```

Comment:
the `if` should start on a newline ideally but no big deal

Reply:
I've started using elm-format in my editor, but when I tried that on one of your text files, it changed it significantly, so I'm not using it there. And I haven't internalized all the rules.

Would you like me to push a little change to add the newline?